lib/MusicBrainz/Server/Edit/Historic/AddDiscID.pm

```diff
@@ -8,7 +8,7 @@ use MusicBrainz::Server::Translation qw( N_l );
 
 use MusicBrainz::Server::Edit::Historic::Base;
 
-sub edit_name     { N_l('Add disc ID') }
+sub edit_name     { N_l('Add medium ID') }
```

Comment:
No, this is really something specific to CDs. See https://wiki.musicbrainz.org/Disc_ID